README.md

```diff
@@ -1,6 +1,14 @@
 # refinebio-processor-documentation
 For tracking documentation of the methodology behind our processor tests: https://github.com/AlexsLemonade/refinebio/issues/2470
 
+## New Required Tests
+
+In addition to improving the existing tests as listed in the links below, we also would like to run a few tests on a weekly basis to make sure things aren't drifting without us knowing.
+These can be configured to run as cron jobs on the foreman server.
+These should check:
+- that brainarray has not drastically changed genes since the last time it was run
```

Comment:
Similar to what checking about genes for BrainArray does for our Affymetrix microarray samples, we could also consider checking the genes (or transcripts) that show up in different transcriptome indices that affect our RNA-seq samples.